crates/bevy_window/src/window.rs

```diff
@@ -169,53 +171,45 @@ pub struct Window {
     pub canvas: Option<String>,
     command_queue: Vec<WindowCommand>,
 }
-
+/// A command to be sent to a window.
 #[derive(Debug)]
 pub enum WindowCommand {
```

Comment:
IMO we should have an example in a doc test here. Use no_run; this is just to verify that it compiles and demonstrate usage.

Reply:
Bevy apps don't interact with `WindowCommand` directly. Instead, they use the methods on the `Window` `struct`. This `enum` is for crates that actually implement Bevy's windowing functionality (like `bevy_winit`). `bevy_window` is, as mentioned in an earlier commit, only a common interface for windowing functionality, so I don't think we need a doc test here.

Reply:
I'm going to clarify this in a future commit.